### PR TITLE
Added weatherforyou.net to display weather images.

### PR DESCRIPTION
**Pull request by kragger**
Added Country and usewttrin variables.  weatherforyou takes a country variable and usewttrin set to 'yes' will return weather output to original settings

### CODE REVIEW

File: server/routes/config.api.js [hunk 1]
```diff
@@ -182,6 +182,8 @@ if (Object.keys(json).length !== 0) {
   config_current.output = json.output || config_current.output;
   config_current.city = json.city || config_current.city;
   config_current.state = json.state || config_current.state;
+  config_current.country = json.country || config_current.country;
```
Comment (liam8888999) [inline server/routes/config.api.js:185]: These 2 are unneccesry, only for importing variables from the old bash version config <v1.0 a list of them can be found here https://github.com/liam8888999/ErsatzTV-Filler/blob/bash-version/sample-config.conf and if they are not in there it cant be translated to json format config anyway

**Comment by liam8888999** [comment]
Also need to add the variables to https://github.com/liam8888999/ErsatzTV-Filler/blob/5ccf5e68469cfa45263a171f52b17f16ba3afe5b/server/modules/config-loader.module.js#L83 so they get added to peoples config. other than that without testing it is looking fairly solid

**Comment by liam8888999** [comment]
there is a contributor section on the updates page too if you wanted to add yourself

File: server/routes/config.api.js (hunk 1 above)
Comment (kragger) [reply server/routes/config.api.js]: Removed

**Comment by kragger** [comment]
> Also need to add the variables to
> 
> https://github.com/liam8888999/ErsatzTV-Filler/blob/5ccf5e68469cfa45263a171f52b17f16ba3afe5b/server/modules/config-loader.module.js#L83
> 
> so they get added to peoples config. other than that without testing it is looking fairly solid
Added the variables here

**Comment by liam8888999** [comment]
closed due to being superceded by #78

**Pull request closed without merging**